ts/component/table-builder.ts

```diff
@@ -226,6 +226,11 @@ export class TableBuilder<ParentType, ChildType> {
       this.addFilterControls(topControlsContainer);
       this.addPagingControls(topControlsContainer);
       this.container.appendChild(topControlsContainer);
+      if (this.definition.bulkActions || this.definition.staticActions) {
+        topControlsContainer.className =
+          "flex justify-end mt-4 items-top space-x-2";
+        this.addActionButtons(topControlsContainer);
+      }
```

Comment:
These should be displayed as similar to the ones on the bottom as possible

* include even when `disablePageControls` is `true`
* display left of the paging controls if present. The top paging controls include more than the bottom and these should not be split up, so it still won't look the exact same.